jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -84,30 +86,48 @@ protected void startNotificationThread() {
 				return thread;
 			};
 
-			Thread notifier = AccessController.doPrivileged(createThread);
+			Thread notifier = java.security.AccessController.doPrivileged(createThread);
 			notifier.start();
 		}
 	}
 
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
 	 */
-	/*[IF Sidecar19-SE]
+	/*[IF JAVA_SPEC_VERSION > 8]*/
 	@Deprecated(forRemoval = true, since = "1.8")
-	/*[ELSE]*/
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
 	@Deprecated
-	/*[ENDIF]*/
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemory() {
+		/*[IF JAVA_SPEC_VERSION >= 14]*/
+		return osinstance.getTotalMemorySize();
+		/*[ELSE] JAVA_SPEC_VERSION >= 14 */
 		return osinstance.getTotalPhysicalMemorySize();
+		/*[ENDIF] JAVA_SPEC_VERSION >= 14 */
 	}
 
+	/*[IF JAVA_SPEC_VERSION > 8]*/
+	@Deprecated(forRemoval = true, since = "1.8")
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
+	@Deprecated
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemorySize() {
+		/*[IF JAVA_SPEC_VERSION >= 14]*/
+		return osinstance.getTotalMemorySize();
+		/*[ELSE] JAVA_SPEC_VERSION >= 14 */
 		return osinstance.getTotalPhysicalMemorySize();
+		/*[ENDIF] JAVA_SPEC_VERSION >= 14 */
 	}
 
 	/**
 	 * TODO Was this intended to be exposed in com.ibm.lang.management.MemoryMXBean?
```

Comment:
Since this is deprecated for removal, the TODO can be removed.

Reply:
Removed.